src/messages/MessageElement.hpp

```diff
@@ -87,54 +95,64 @@ enum class MessageElementFlag : int64_t {
     // - Chatterino top donator badge
     // - Chatterino special pepe badge
     // - Chatterino gnome badge
-    BadgeChatterino = (1LL << 18),
-
-    // Slot 7: FrankerFaceZ
+    BadgeChatterino = (1LL << 20),
+
+    // Slot 7: Seventv
+    // - Seventv Admin
+    // - Seventv Dungeon Mistress
+    // - Seventv Moderator
+    // - Seventv Subscriber
+    // - Seventv Translator
+    // - Seventv Contributor
+    BadgeSeventv = (1LL << 21),
+
+    // Slot 8: FrankerFaceZ
     // - FFZ developer badge
     // - FFZ bot badge
     // - FFZ donator badge
-    BadgeFfz = (1LL << 19),
+    BadgeFfz = (1LL << 22),
 
     Badges = BadgeGlobalAuthority | BadgePredictions | BadgeChannelAuthority |
-             BadgeSubscription | BadgeVanity | BadgeChatterino | BadgeFfz,
+             BadgeSubscription | BadgeVanity | BadgeChatterino | BadgeSeventv |
+             BadgeFfz,
 
-    ChannelName = (1LL << 20),
+    ChannelName = (1LL << 23),
 
-    BitsAmount = (1LL << 21),
+    BitsAmount = (1LL << 24),
 
-    ModeratorTools = (1LL << 22),
+    ModeratorTools = (1LL << 25),
 
-    EmojiImage = (1LL << 23),
-    EmojiText = (1LL << 24),
+    EmojiImage = (1LL << 26),
+    EmojiText = (1LL << 27),
     EmojiAll = EmojiImage | EmojiText,
 
-    AlwaysShow = (1LL << 25),
+    AlwaysShow = (1LL << 28),
 
     // used in the ChannelView class to make the collapse buttons visible if
     // needed
-    Collapsed = (1LL << 26),
+    Collapsed = (1LL << 29),
 
     // used for dynamic bold usernames
-    BoldUsername = (1LL << 27),
-    NonBoldUsername = (1LL << 28),
+    BoldUsername = (1LL << 30),
+    NonBoldUsername = (1LL << 31),
 
     // for links
-    LowercaseLink = (1LL << 29),
-    OriginalLink = (1LL << 30),
+    LowercaseLink = (1LL << 32),
+    OriginalLink = (1LL << 33),
 
     // ZeroWidthEmotes are emotes that are supposed to overlay over any pre-existing emotes
-    // e.g. BTTV's SoSnowy during christmas season
-    ZeroWidthEmote = (1LL << 31),
+    // e.g. BTTV's SoSnowy during christmas season or zerowidth 7TV emotes
```

Comment:
```suggestion
    // e.g. BTTV's SoSnowy during christmas season or 7TV's RainTime
```
"zerowidth emotes are 7tv zerowidth emotes" (yes I'm aware it's a chatterino7 comment but I've wanted it fixed for a while.)